src/server.ts

```diff
@@ -25,6 +26,9 @@ export class InversifyExpressServer  {
     ) {
         this._container = container;
         this._router = customRouter || express.Router();
+        this._routingConfig = {
+            rootPath: "/"
```

Comment:
Yeah, that sounds good to me. Besides there is no need for `setRoutingConfig` in this approach :ok_hand: